2019-06-21 10:13:54,648|ERROR|main|Please input the config path.
2019-06-21 10:13:56,588|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-21 10:13:56,591|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-21 10:13:56,775|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-21 10:13:56,775|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-21 10:13:56,775|ERROR|main|****************** All Component has started successfully.******************
2019-06-21 10:13:56,775|ERROR|main|UploadServer has started successfully, taken 2135 millis.
2019-06-21 10:35:29,966|ERROR|main|Please input the config path.
2019-06-21 10:35:31,446|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-21 10:35:31,449|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-21 10:35:31,632|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-21 10:35:31,633|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-21 10:35:31,633|ERROR|main|****************** All Component has started successfully.******************
2019-06-21 10:35:31,633|ERROR|main|UploadServer has started successfully, taken 1676 millis.
2019-06-21 10:41:55,588|ERROR|qtp1860491691-20|查看服务器日志异常...
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.web.servlet.serverlist.GetServerLogServlet.execute(GetServerLogServlet.java:68) ~[bin/:na]
	at com.upload.web.servlet.BaseHandlerServlet.doPost(BaseHandlerServlet.java:93) [bin/:na]
	at com.upload.web.servlet.BaseHandlerServlet.doGet(BaseHandlerServlet.java:52) [bin/:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:687) [servlet-api-3.1.jar:3.1.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790) [servlet-api-3.1.jar:3.1.0]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:838) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:543) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1584) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1228) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:168) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:481) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1553) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:166) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1130) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:52) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.Server.handle(Server.java:564) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:318) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:112) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:672) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:590) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]
2019-06-21 13:51:24,276|ERROR|main|Please input the config path.
2019-06-21 13:51:25,051|ERROR|pool-1-thread-1|定时任务:刷新服务器信息开始
2019-06-21 13:51:25,135|ERROR|pool-1-thread-1|定时任务:刷新服务器信息结束
2019-06-21 13:51:25,642|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-21 13:51:25,642|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-21 13:51:25,790|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-21 13:51:25,790|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-21 13:51:25,790|ERROR|main|****************** All Component has started successfully.******************
2019-06-21 13:51:25,790|ERROR|main|UploadServer has started successfully, taken 1521 millis.
2019-06-21 13:56:27,446|ERROR|pool-1-thread-1|定时任务:检查服务器运行状态开始!
2019-06-21 13:56:43,910|ERROR|pool-1-thread-1|定时任务:检查服务器运行状态开始!
2019-06-21 13:58:04,968|ERROR|pool-1-thread-1|定时任务:检查服务器运行状态开始!
2019-06-21 13:58:21,802|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 13:58:21,803|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 13:58:22,812|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 13:58:22,813|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 13:58:23,836|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 13:58:23,838|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 13:58:24,017|ERROR|pool-1-thread-1|******连接SSH服务器成功：192.168.1.96:5188,root
2019-06-21 13:58:24,921|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 13:58:25,011|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 13:58:25,012|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 13:58:25,087|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 13:58:25,088|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 13:58:25,194|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 13:58:25,194|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 13:58:26,206|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 13:58:26,207|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 13:58:43,385|ERROR|pool-1-thread-1|定时任务:检查服务器运行状态结束!
2019-06-21 13:58:43,386|ERROR|pool-1-thread-1|定时任务:刷新服务器信息开始
2019-06-21 13:58:43,455|ERROR|pool-1-thread-1|定时任务:刷新服务器信息结束
2019-06-21 14:02:43,904|ERROR|main|Please input the config path.
2019-06-21 14:02:44,697|ERROR|pool-1-thread-1|定时任务:刷新服务器信息开始
2019-06-21 14:02:44,785|ERROR|pool-1-thread-1|定时任务:刷新服务器信息结束
2019-06-21 14:02:44,785|ERROR|pool-1-thread-1|定时任务:检查服务器运行状态开始!
2019-06-21 14:02:45,332|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-21 14:02:45,332|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-21 14:02:45,502|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-21 14:02:45,502|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-21 14:02:45,502|ERROR|main|****************** All Component has started successfully.******************
2019-06-21 14:02:45,503|ERROR|main|UploadServer has started successfully, taken 1604 millis.
2019-06-21 14:02:46,036|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:02:46,037|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:02:47,047|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:02:47,049|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:02:48,074|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:02:48,074|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:02:48,272|ERROR|pool-1-thread-1|******连接SSH服务器成功：192.168.1.96:5188,root
2019-06-21 14:02:49,169|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:02:49,297|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:02:49,297|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:02:49,387|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:02:49,387|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:02:49,482|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:02:49,482|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:02:50,487|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:02:50,487|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:03:03,396|ERROR|pool-1-thread-1|定时任务:检查服务器运行状态结束!
2019-06-21 14:03:33,398|ERROR|pool-1-thread-1|定时任务:检查服务器运行状态开始!
2019-06-21 14:03:34,408|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:03:34,409|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:03:35,415|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:03:35,417|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:03:36,430|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:03:36,430|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:03:36,574|ERROR|pool-1-thread-1|******连接SSH服务器成功：192.168.1.96:5188,root
2019-06-21 14:03:37,457|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:03:37,571|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:03:37,571|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:03:37,648|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:03:37,648|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:03:37,771|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:03:37,772|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:03:38,779|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:03:38,780|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:04:41,240|ERROR|pool-1-thread-1|定时任务:检查服务器运行状态开始!
2019-06-21 14:04:42,251|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:04:42,252|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:04:43,254|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:04:43,254|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:04:44,271|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:04:44,273|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:04:44,370|ERROR|pool-1-thread-1|******连接SSH服务器成功：192.168.1.96:5188,root
2019-06-21 14:04:45,263|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:04:45,372|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:04:45,372|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:04:45,464|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:04:45,464|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:04:45,587|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:04:45,587|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:04:46,582|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(Unknown Source) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:04:46,583|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:04:51,907|ERROR|pool-1-thread-1|定时任务:检查服务器运行状态结束!
2019-06-21 14:05:21,908|ERROR|pool-1-thread-1|定时任务:检查服务器运行状态开始!
2019-06-21 14:05:22,919|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:05:22,920|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:05:23,931|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:05:23,931|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:05:24,937|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:05:24,937|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:05:25,048|ERROR|pool-1-thread-1|******连接SSH服务器成功：192.168.1.96:5188,root
2019-06-21 14:05:25,941|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:05:26,044|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:05:26,045|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:05:26,132|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:05:26,132|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:05:26,229|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:519) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-21 14:05:26,229|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

2019-06-21 14:05:27,241|ERROR|pool-1-thread-1|
com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused: connect
	at com.jcraft.jsch.Util.createSocket(Util.java:394) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:215) ~[jsch-0.1.55.jar:na]
	at com.jcraft.jsch.Session.connect(Session.java:183) ~[jsch-0.1.55.jar:na]
	at com.upload.util.JSchUtils.connectWithKey(JSchUtils.java:102) ~[bin/:na]
	at com.upload.component.ServerListComponent.checkServerState(ServerListComponent.java:94) [bin/:na]
	at com.upload.component.ServerListComponent.lambda$3(ServerListComponent.java:68) [bin/:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_60]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_60]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_60]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_60]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method) ~[na:1.8.0_60]
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_60]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_60]
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172) ~[na:1.8.0_60]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_60]
	at java.net.Socket.connect(Socket.java:538) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:434) ~[na:1.8.0_60]
	at java.net.Socket.<init>(Socket.java:211) ~[na:1.8.0_60]
	at com.jcraft.jsch.Util$1.run(Util.java:362) ~[jsch-0.1.55.jar:na]
	... 1 common frames omitted
2019-06-21 14:05:27,242|ERROR|pool-1-thread-1|******关闭SSH服务器的连接！

